# central positions of balls
#位置:ball_detector/ball_positions.msg

Header header   # dependency: std_msgs
geometry_msgs/Point[] positions     # dependency: sensor_msgs
float32 front_left_speed
float32 front_right_speed
float32 back_left_speed
float32 back_right_speed
